extensions/amp-ad/amp-ad.md

```diff
@@ -21,7 +21,7 @@ limitations under the License.
 <table>
   <tr>
     <td class="col-fourty"><strong>Description</strong></td>
-    <td>A container to display an ad. AMP documents only support ads served via HTTPS.</td>
+    <td>A container to display an ad. Use <code>amp-embed</code> when semantically more accurate. AMP documents only support ads/embeds served via HTTPS.</td>
```

Comment:
Might want to add a one liner for along the line of : "The `<amp-embed>` is an alias to the [`<amp-ad>`] tag, deriving all of its functionality with a different tag name.'